web/server/codechecker_server/migrations/report/versions/6cb6a3a41967_system_comments.py

```diff
@@ -0,0 +1,28 @@
+"""System comments
+
+Revision ID: 6cb6a3a41967
+Revises: 3e91d0612422
+Create Date: 2019-04-02 16:12:46.794131
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '6cb6a3a41967'
+down_revision = '3e91d0612422'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('comments', sa.Column('kind', sa.Enum(u'user',
```

Comment:
It will fail on postgresql. We can fix this by using the following instructions: https://stackoverflow.com/questions/37848815/sqlalchemy-postgresql-enum-does-not-create-type-on-db-migrate.
```py
comment_kind_name = 'comment_kind'
enum_values = (u'user', u'system')

commnet_kind_column = sa.Column('kind', sa.Enum(*enum_values,
                                                name=comment_kind_name),
                                server_default=u'user',
                                nullable=False)
def upgrade():
    ctx = op.get_context()
    dialect = ctx.dialect.name

    if dialect == 'postgresql':
        from sqlalchemy.dialects import postgresql
        comment_kind = postgresql.ENUM(*enum_values, name=comment_kind_name)
        comment_kind.create(op.get_bind())

        op.add_column('comments', commnet_kind_column)
    elif dialect == 'sqlite':
        op.add_column('comments', commnet_kind_column)
def downgrade():
    op.drop_column('comments', 'kind')

    ctx = op.get_context()
    dialect = ctx.dialect.name

    if dialect == 'postgresql':
        from sqlalchemy.dialects import postgresql
        comment_kind = postgresql.ENUM(*enum_values, name=comment_kind_name)
        comment_kind.drop(op.get_bind())
```

But as we discussed it personally we will use integers instead of enums to avoid this kind of hacks.